brave/src/test/java/brave/features/handler/CountingChildrenTest.java

```diff
@@ -103,7 +104,7 @@
     assertThat(spans)
       .extracting(Span::name, s -> s.tags().get("childCount"))
       .containsExactly(
-        tuple("root1", "0"), tuple("root1child1", "0"), tuple("root1child2", "0")
+        tuple("root1", "2"), tuple("root1child1", "0"), tuple("root1child2", "0")
```

Comment:
yay we can implement `child_span_count` in the future with a native brave SpanHandler for stackdriver

cc @meltsufin @saturnism

https://cloud.google.com/trace/docs/reference/v2/rpc/google.devtools.cloudtrace.v2#google.devtools.cloudtrace.v2.Span

Reply:
neat!